torch_xla/csrc/runtime/stablehlo_helper.cc

```diff
@@ -22,7 +22,9 @@ static std::string getHloModuleStr(const xla::HloModuleProto* proto) {
 static std::string getMlirModuleStr(mlir::ModuleOp& mlir_module) {
   std::string txt_mlir_module;
   llvm::raw_string_ostream os{txt_mlir_module};
-  mlir_module.print(os);
+  // Enable Debug Info to include source line info in the StableHLO dump.
+  mlir_module.print(
```

Comment:
I guess if we don;t enable `XLA_HLO_DEBUG=1` this will be a no-op? I am trying to think whether we need another flag like `XLA_SHLO_DEBUG` but that seems to be an overkill.

Reply:
Yes, that makes sense.

I think we should turn on the debug info in MLIR Module dump when `XLA_HLO_DEBUG=1`. I don't think we need another flag, since there isn't a use case that we want HLO to have meta data but do not want those in StableHLO : )